presentation-api/index.html

```diff
@@ -97,6 +104,15 @@
   }
 });
 
+document.querySelector('#send').addEventListener('click', () => {
+  if (presentationConnection && presentationConnection.state == 'connected') {
+    const message = document.querySelector('#message').value.trim();
+    const lang = document.body.lang || 'en-US';
+    ChromeSamples.log('Sending ' + message + ' to ' + presentationConnection.id);
+    presentationConnection.send(JSON.stringify({string: message, lang: lang}));
```

Comment:
The issue is that the lang attribute affects font rendering in CJK.  See https://github.com/w3c/presentation-api/issues/218